### DISCUSSION TITLE
Custom DoH Question

### DISCUSSION THREAD

**Discussion by DukeJukemEG** [HotCakeX/WinSecureDNSMgr#9] [Q&A]
Hi :D.  I'm trying to use Control D as my DNS and it works fine but I noticed when I set it I'm given a message stating "Using the main CloudFlare Encrypted API to resolve."

I was curious if it was possible to have it say "Using the main Quad9 Encrypted API to resolve." I don't like CloudFlare due to a few things I've read and their affiliation with Mozilla, which is also making questionable decisions lately so...yea.

Amazing work you've done here and your Harden Windows Security.

**Comment by ghost** [comment]
Did you ever find an answer?

**Comment by HotCakeX** [comment]
Hi,
I'm sorry for not responding, i can implement a feature where you will choose the resolver. I can implement it in the same user interface as Harden Windows Security module too so you will have a GUI.
